lib/core/src/server/config/webpack.config.default.js

```diff
@@ -19,16 +24,18 @@ export function createDefaultWebpackConfig(storybookBaseConfig) {
             },
             {
               loader: require.resolve('postcss-loader'),
-              options: {
-                ident: 'postcss', // https://webpack.js.org/guides/migrating/#complex-options
-                postcss: {},
-                plugins: () => [
-                  require('postcss-flexbugs-fixes'), // eslint-disable-line
-                  autoprefixer({
-                    flexbox: 'no-2009',
-                  }),
-                ],
-              },
+              options: postcssConfig
+                ? {
+                    postcssConfig,
+                  }
+                : {
+                    plugins: () => [
+                      require('postcss-flexbugs-fixes'), // eslint-disable-line global-require
```

Comment:
I understand that something won't work if this plugin will be missing?

Reply:
The plugins are currently available, correct? I didn't remove dependencies for that reason.

Reply:
I mean, should we somehow merge it to the user's config?